frontend/public/components/events.jsx

```diff
@@ -367,18 +367,17 @@ class EventStream extends React.Component {
       })
       .onopen(() => {
         this.messages = {};
-        this.setState({ error: false, loading: false, sortedMessages: [], filteredEvents: [] });
+        this.setState({ error: false, loading: false });
       })
       .onclose((evt) => {
         if (evt && evt.wasClean === false) {
           this.setState({ error: evt.reason || 'Connection did not close cleanly.' });
         }
         this.messages = {};
-        this.setState({ sortedMessages: [], filteredEvents: [] });
```

Comment:
Why not remove the messages `onclose()`?

Reply:
what happen when I tried to repruduce the bug:
a - close
b - open
so to prevent the flicker I needed to remove the list clear from open and close ...

Reply:
so the flicker happens because a new set of events is overriding the old one.
maybe theres a better way to handle this to avoid multiple fetches in a short period of time, but I guess not for this PR

Reply:
hmm, you mean the `this.messages = {};` :+1: removing it.